backend/dal/Services/Concrete/UserService.cs

```diff
@@ -64,6 +64,19 @@ public User Activate()
             this.Logger.LogInformation($"User Activated: '{id}' - '{display_name}'.");
             return entity;
         }
+
+        /// <summary>
+        /// Activate the new authenticated user with the PIMS datasource.
+        /// </summary>
+        /// <param name="request"></param>
+        /// <returns></returns>
+        public AccessRequest AddAccessRequest(AccessRequest request)
+        {
+            if (request == null) throw new ArgumentNullException();
+            this.Context.AccessRequests.Add(request);
```

Comment:
Should add the `CreatedById` to the entity.

Reply:
I thought we had some of those base entity fields being filled in automatically, is created by not on that list? I feel like it should be.

Reply:
I see, that only applies to the service type overriding the base class.

How would you feel about extending the commit transaction method to save this information for us? perhaps not with this PR, but something to consider.

Reply:
I would prefer to move it to the transaction method.  It requires a bit of extra leg work, as we need every object that is being submitted for an add/update.  I'll look into this later for Tech-debt